### PR TITLE
Tests improving #2

### PR DESCRIPTION
**Pull request by Staslgvin23**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1,264 +1,618 @@
...
+  it('should display correct value for 30', function () {
+    expect(number2String(30)).to.equal('тридцять гривень нуль копійок');
+  });
+
+  it('should display correct value for 41', function () {
+    expect(number2String(41)).to.equal('сорок одна гривня нуль копійок');
+  });
+
+  it('should display correct value for 52', function () {
+    expect(number2String(52)).to.eql('п\'ятдесят дві гривні нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: Please, use `to.equal` instead of `to.eql`.

File: test/index.js [hunk 2]
```diff
@@ -1,264 +1,618 @@
...
+  });
+
+  it('should display correct value for 65', function () {
+    expect(number2String(65)).to.have.string('шістдесят п\'ять гривень нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: The same here.

File: test/index.js [hunk 3]
```diff
@@ -1,264 +1,618 @@
...
+  });
+
+  it('should display correct value for 70', function () {
+    expect(number2String(70)).to.have.string('сімдесят гривень нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js [hunk 4]
```diff
@@ -1,264 +1,618 @@
...
+  });
+
+  it('should display correct value for 81', function () {
+    expect(number2String(81)).to.have.string('вісімдесят одна гривня нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js [hunk 5]
```diff
@@ -1,264 +1,618 @@
...
+  });
+
+  it('should display correct value for 92', function () {
+    expect(number2String(92)).to.eql('дев\'яносто дві гривні нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js [hunk 6]
```diff
@@ -1,264 +1,618 @@
...
+  it('should display correct value for 100', function () {
+    expect(number2String(100)).to.equal('сто гривень нуль копійок');
+  });
+
+  it('should display correct value for 101', function () {
+    expect(number2String(101)).to.equal('сто одна гривня нуль копійок');
+  });
+
+  it('should display correct value for 102', function () {
+    expect(number2String(102)).to.eql('сто дві гривні нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js [hunk 7]
```diff
@@ -1,264 +1,618 @@
...
+  });
+
+  it('should display correct value for 105', function () {
+    expect(number2String(105)).to.equal('сто п\'ять гривень нуль копійок');
+  });
+
+  it('should display correct value for 110', function () {
+    expect(number2String(110)).to.eql('сто десять гривень нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js [hunk 8]
```diff
@@ -1,264 +1,618 @@
...
+  it('should display correct value for 120', function () {
+    expect(number2String(120)).to.equal('сто двадцять гривень нуль копійок');
+  });
+
+  it('should display correct value for 131', function () {
+    expect(number2String(131)).to.equal('сто тридцять одна гривня нуль копійок');
+  });
+
+  it('should display correct value for 142', function () {
+    expect(number2String(142)).to.eql('сто сорок дві гривні нуль копійок');
```
Comment (vshashkov) [inline test/index.js]: And here.

File: test/index.js (hunk 1 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 2 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 8 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 7 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 6 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 5 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 4 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

File: test/index.js (hunk 3 above)
Comment (Staslgvin23) [reply test/index.js]: Done.

**Review by vshashkov — APPROVED** [review]

**Pull request merged**